src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let person = people.filter(man => man.sex === 'm');
+
+  person = century
+    ? person.filter(boy => Math.ceil(boy.died / 100) === century)
+    : person;
+
+  const averageSum = person.reduce(
+    (sum, { died, born }) => sum + (died - born), 0
+  );
```

Comment:
comment 'move it into function and reuse in other functions as well' is still actual.
it's a duplicate of code on 57-59 lines